go/chat/sender.go

```diff
@@ -1198,6 +1218,13 @@ func (s *BlockingSender) Send(ctx context.Context, convID chat1.ConversationID,
 			s.Debug(ctx, "Send: error in Prepare: %s", err)
 			return nil, nil, err
 		}
+		if !clearedCache && prepareRes.HasBlankTopicName {
```

Comment:
Does this work right with non-team convs?

Reply:
fixed to just apply to TEAM type